bundles/AdminBundle/Controller/Admin/LoginController.php

```diff
@@ -286,4 +288,29 @@ public function twoFactorAuthenticationAction(Request $request, BruteforceProtec
     public function twoFactorAuthenticationVerifyAction(Request $request)
     {
     }
+
+    /**
+     * @return bool
+     */
+    public function detectBrowser()
+    {
+        $supported      = false;
```

Comment:
create a twig extension that does that?

Reply:
quite specific for login request...what is the benefit?

Reply:
I think it's more elegant...

Reply:
I think it's fine as it is ...